.travis.yml

```diff
@@ -144,6 +142,8 @@ jobs:
     - php: 5.6
       env: WP_VERSION=4.5
       if: NOT commit_message =~ js_only AND NOT commit_message =~ dependabot
+    - php: 5.5
+      if: NOT commit_message =~ js_only AND NOT commit_message =~ dependabot
```

Comment:
you need to include `env: WP_VERSION=4.5` if you only want to test against wp 4.5.

Also, I think we might as well just have a single PHP5.5 and WP 4.5 job.  I think also testing php5.6 on 4.5 is unnecessary.  The biggest changes in php happened going from PHP 5.3->5.4 and 5.6->7.0.  Php 5.6 and 5.5 are _very slow_ builds for our test suite so whatever we can eliminate will help speed up test runs.

Reply:
makes sense... will change